stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/MonitoredMethodRequest.java

```diff
@@ -1,5 +1,6 @@
 package org.stagemonitor.requestmonitor;
 
+import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
Ok, that complicates matters. I'll need to recreate the ListenableFuture (or something simplier). You already have a CompletedFuture class, so I can put it along side that.